scripts/import/import_hgmd.pl

```diff
@@ -336,6 +347,18 @@ sub get_variation_set_id {
   }
 }
 
+sub get_attrib_id {
+  my ($type, $value) = @_;
+
+  my $aa = $vdb2->get_AttributeAdaptor();
```

Comment:
Not relevant for PR review but why is it called $vdb2? Where is $vdb1?

Reply:
Don't know. There was never a $vdb1 in this script, it has always been $vdb2 since the start of the script 7 years ago.